webapp/src/App.tsx

```diff
@@ -125,6 +127,11 @@ const App: FC = () => {
             ]);
         }
 
+        if (isMigrating)
+        {
+            setAppState(AppState.ProbeForBackend);
+        }
+    
         // eslint-disable-next-line react-hooks/exhaustive-deps
     }, [instance, inProgress, isAuthenticated, appState]);
```

Comment:
isMigrating needs to be part of the dependency array.
If it's causing an infinite loop, try changing the check on line 130 to 
`if (isMigrating && appState !== AppState.ProbeForBackend)`

```suggestion
    }, [instance, inProgress, isAuthenticated, appState, isMigrating]);
```